test/electionguard/generators/manifest.hpp

```diff
@@ -11,21 +11,34 @@
 using namespace electionguard;
 using namespace std;
 
-namespace electionguard::test::mocks
+#ifndef TEST_SPEC_VERSION
+#    define TEST_SPEC_VERSION "0.95.0"
+#endif
+
+#ifndef TEST_USE_SAMPLE
+#    define TEST_USE_SAMPLE "hamilton-general"
```

Comment:
should probably be runtime configurable and not compile time configurable

Reply:
Same for the `election_manifest_jefferson_county.json` below?